content/general/concepts/github/github.md

```diff
@@ -0,0 +1,17 @@
+---
+Title: 'Github'
+Description: 'Github is a UI based version control that uses git.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'GitHub'
+  - 'Git'
+CatalogContent:
+  - "learn-git"
+  - "paths/learn-how-to-build-websites"
+---
+
+## Github
+
```

Comment:
```suggestion
```

I don't think this header is necessary since it will already be displayed via the meta-`Title`.